code.js

```diff
@@ -1,7 +1,8 @@
 function sum(a) {
-    var sum = a[0];
+    var sum = 0;
     for(var i = 0; i < a.length; i++) {
         sum += a[i];
     }
+    console.log(sum);
```

Comment:
Please don't leave debug code in your submission.

Reply:
removed debugging code. apologies